### ISSUE TITLE
is DEPRECATED doesn't work on Routines in precomped modules

### ISSUE THREAD

**Issue by lizmat** [rakudo/rakudo#1219]
Foo.pm:
-------------------
```
class A {
    method a() is DEPRECATED<b> {
    }
}
```
-------------------
```bash
$ perl6 --ll-exception -I. -MFoo -e 'dd A.a'
```
```
Cannot invoke this object (REPR: Null; VMNull)
   at SETTING::src/core/traits.pm:150  (/Users/liz/Github/rakudo.moar/install/share/perl6/runtime/CORE.setting.moarvm:)
 from /Users/liz/Github/rakudo.moar/Foo.pm6 (Foo):3  (/Users/liz/Github/rakudo.moar/.precomp/E7148DC9FED15B480AF079994892987152D023CD.1509221700.44536/20/201A6B3053CC1422D2C3670B62616221D2290929:a)
 from -e:1  (<ephemeral file>:<unit>)
 from -e:1  (<ephemeral file>:<unit-outer>)
```

**Comment by skids** [comment]
Workaround PRd in https://github.com/rakudo/rakudo/pull/1861

**Comment by lizmat** [comment]
Fixed with 472f6e4844

**Comment by vrurg** [comment]
Looks like merge Raku/roast#546 closed this ticket.
